core/trino-main/src/main/java/io/trino/sql/relational/SpecialForm.java

```diff
@@ -67,6 +67,7 @@ public Form getForm()
         return form;
     }
 
+    @JsonProperty
```

Comment:
Add a test in `io.trino.sql.routine.TestSqlFunctions`

Reply:
Done. Without the @JsonProperty added test will fail.